clang/docs/analyzer/checkers.rst

```diff
@@ -1766,13 +1770,6 @@ are assumed to succeed.)
    fclose(p);
  }
 
-**Limitations**
-
-The checker does not track the correspondence between integer file descriptors
-and ``FILE *`` pointers. Operations on standard streams like ``stdin`` are not
-treated specially and are therefore often not recognized (because these streams
-are usually not opened explicitly by the program, and are global variables).
```

Comment:
I wasn't sure of this section, so I just removed it. Let me know if it's still relevant or not.

Reply:
Hmm, perhaps keep a note saying "This checker does not track operations that use integer file descriptors instead of `FILE *` pointers". (If I recall correctly this is true and might be relevant for someone who's used to work with low-level stuff.) However it's also completely fine (from my POV) if you delete the whole Limitations section.

Reply:
Yea, but I didn't know exactly what it refers to. We could add an example for such a case there. That would help.

Reply:
It refers to functions like `write` [(man page)](https://www.man7.org/linux/man-pages/man2/write.2.html) where the file is specified by an `int` argument.

Reply:
Ah indeed. So basically opening a file with file descriptor "ints" and then using `fdopen()` to get the corresponding `FILE *` - and back with `fileno()`. Yea, that seems weird to do :D Probably has its use somewhere though.
Anyways, makes sense now.